README.md

```diff
@@ -94,64 +94,14 @@ twig:
 #### Required
 
 First, you have to select your language, other settings are optional (see below).
+If you don't have this file you can create one.
```

Comment:
Not sure if it's necessary

Reply:
@jmsche I didn't have this file but configuration was passed here, so I added this.